src/PrestaShopBundle/Resources/views/Admin/Improve/International/Locations/Country/Blocks/format_actions.html.twig

```diff
@@ -0,0 +1,47 @@
+{% set confirmation = 'Are you sure you want to restore the default address format for this country?'|trans({},'Admin.International.Feature') %}
+<div class="col-8 address-format-actions">
+  <span title=""
+    data-toggle="tooltip"
+    class="label-tooltip"
+    data-original-title="{{ 'This will restore your last registered address format.'|trans({}, 'Admin.International.Help') }}"
+    data-html="true" data-placement="top">
+    <a id="useLastDefaultLayout" href="javascript:void(0)"
+       data-confirmation="{{ confirmation }}"
+       data-default-layout="{{ encodingAddressLayout }}"
+       class="btn btn-outline-secondary reset-address-layout">
+      {{ 'Use the last registered format'|trans({}, 'Admin.International.Feature') }}
+    </a>
+  </span>
+  <span title="" data-toggle="tooltip" class="label-tooltip"
+    data-original-title="{{ 'This will restore the default address format for this country'|trans({}, 'Admin.International.Help') }}"
+    data-html="true">
+        <a id="useDefaultLayoutSystem"
+           href="javascript:void(0)"
+           data-confirmation="{{ confirmation }}"
+           data-default-layout="{{ defaultLayout }}"
+           class="btn btn-outline-secondary reset-address-layout">
+          {{ 'Use the default format'|trans({}, 'Admin.International.Feature') }}
+        </a>
+  </span>
+  <span title="" data-toggle="tooltip" class="label-tooltip"
+        data-original-title="{{ 'This will restore your current address format.'|trans({}, 'Admin.International.Help') }}"
```

Comment:
Well, I'm not sure about the wording here, it seems weird for me to restore something that is already in place.

Reply:
After manually modifying address format value it will be saved and this action will fill text field with saved data after other modifications with given actions were done if any. I agree that it is a bit misleading but it kind of says what it does correctly. I cant think of a valid replacement. Do you have any suggestions?

Reply:
But what I don't get is the difference with the first button _This will restore your last registered address format._, it looks the same to me.
🤔

Reply:
First button takes value from backend and it is the last value submitted. The button in question restores value locally which is saved in browser while editing Country

Reply:
Alright, I think I get it. @marionf has suggested adding a tooltip here to let the users know where to set mandatory fields for customers, here is what I suggest: _Want to set required fields in the customer's registration form? Go to Customers > Addresses and select the information you need._ localized in _Admin.International.Help_.

Reply:
@LouiseBonnard wdyt about: 
_Want to set required fields in the customer's registration form? Go to Customers > Addresses and **set your required fields** ?_

With a link to the page

Reply:
So, we agreed on: _Want to set required fields in the customer's registration form? Go to [1]Customers > Addresses[/1] to configure it._, still in _Admin.International.Help_ and with a redirecting link to bring users to the proper page directly. Would that be possible, @RaimondasSapola?

Reply:
It suits me fine!